### PR TITLE
Filter kit + Title Count JS

### PR DESCRIPTION
**Pull request by kre8sions**
#### Breaking Changes

_Please reflect if your changes may break in some way (changes like removing/renaming props are examples of breaking changes)_

#### Checklist:

- [ ] **SPECS** Please cover your changes with specs
- [ ] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

File: config/data/menu.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+kits:
```
Comment (jasoncypret) [inline config/data/menu.yml]: This file looks out of date

File: spec/pb_kits/playbook/kits/filter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+require_relative "../../../../app/pb_kits/playbook/pb_filter/filter"
+
+RSpec.describe Playbook::PbFilter::Filter do
+  subject { Playbook::PbFilter::Filter }
+
+  it { is_expected.to define_partial }
+
+  # Do not leave this file blank. Use other spec files for example tests.
+end
```
Comment (jasoncypret) [inline spec/pb_kits/playbook/kits/filter_spec.rb]: Don't forget your spec

File: app/pb_kits/playbook/pb_filter/_filter.html.erb [hunk 3]
```diff
@@ -2,5 +2,33 @@
...
+  </div>
+
+  <div class="d-flex align-items-center">
+      <%= pb_rails("caption", props: { text: "sort by:"}) %>
+      <%= pb_rails("button", props: {variant: "link" ,classname: "p-0 ml-3"}) do %>
+          <span>Popularity</span>
+          <%= pb_rails("icon", props: { icon: "sort-amount-down", fixed_width: true }) %>
+      <% end %>
+  </div>
+  </div>
```
Comment (gmfvpereira) [inline app/pb_kits/playbook/pb_filter/_filter.html.erb]: Either indentation is off or there's an extra `</div>`

File: app/pb_kits/playbook/pb_filter/_filter.jsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+const Filter = ({
+  className = 'pb_filter_kit',
+  data,
+  id,
+  children,
+  sortMenu,
+  filters,
+}: FilterProps) => {
+  return (
+    <div
```
Comment (gmfvpereira) [inline app/pb_kits/playbook/pb_filter/_filter.jsx]: If there's nothing other than the return statement, you could omit the curly braces
```
const Filter = ({
  className = 'pb_filter_kit',
  data,
  id,
  children,
  sortMenu,
  filters,
}: FilterProps) => (
  <div
```

**Comment by kre8sions** [comment]
> Is it still wip?

@gmfvpereira Yup still WIP we are releasing rails version first

**Comment by drborges** [comment]
@kre8sions could you add a screenshot to the PR description when you have a chance, please?

**Comment by gmfvpereira** [comment]
I see that there's a wip label in this repo as well, is it possible to use or it has a different meaning?

**Pull request closed without merging**